docs/core/tools/dotnet.md

```diff
@@ -298,7 +298,7 @@ Run an application DLL, such as `myapp.dll`:
 
 `DOTNET_PACKAGES`
 
-The primary package cache. If not set, it defaults to `$HOME/.nuget/packages` on Unix or `%HOME%\NuGet\Packages` on Windows.
+The primary package cache. If not set, it defaults to `$HOME/.nuget/packages` on Unix or `%userprofile%\.nuget\packages` on Windows.
```

Comment:
We should probably change the Unix value as well to match what we have in https://docs.microsoft.com/en-us/nuget/Consume-Packages/managing-the-global-packages-and-cache-folders. @karann-msft can you confirm?

```suggestion
The primary package cache. If not set, it defaults to `~/.nuget/packages` on Unix or `%userprofile%\.nuget\packages` on Windows.
```

Reply:
yup..that's right. Also, it is wrong to call it a _cache_. It should be called "global packages folder".

Reply:
@david1995 can you accept this commit suggestion? When you do that I'll merge this PR. Thanks again!!